Emojigrams/Emojigrams/EmojigramGuessProgramaticLayoutViewController.swift

```diff
@@ -159,6 +160,12 @@ class EmojigramGuessProgramaticLayoutViewController: EmojigramGuesserBaseViewCon
         self.giveUpButton.isHidden = true
     }
     
+    func configurePage() {
+        if true {
+            self.guessTextField.autocorrectionType = .no
```

Comment:
#? what does autocorrectionType do? maybe a comment here?

Reply:
autocorrectionType is a property on UITextFIeld that determines whether autocorrect should be available while the keyboard is in use for that textField. Since the answers might involve nonstandard English words, e.g., Narnia, the app should just let the user enter without autocorrect ruining their answers.